charts/victoria-metrics-k8s-stack/CHANGELOG.md

```diff
@@ -2,16 +2,13 @@
 
 ## Next release
 
-**Release date:** TBD
-
-![AppVersion: **APP_VERSION**](https://img.shields.io/static/v1?label=AppVersion&message=**APP_VERSION**&color=success&logo=)
-![Helm: v3](https://img.shields.io/static/v1?label=Helm&message=v3&color=informational&logo=helm)
-
-* Move `cleanupCRD` option to victoria-metrics-operator chart (#593)
-* Disable `honorTimestamps` for cadvisor scrape job by default (#617)
-* For vmalert all replicas of alertmanager are added to notifiers (only if alertmanager is enabled) (#619)
-* Add `grafanaOperatorDashboardsFormat` option (#615)
-* Fix query expression for memory calculation in `k8s-views-global` dashboard (#636)
+- Move `cleanupCRD` option to victoria-metrics-operator chart (#593)
+- Disable `honorTimestamps` for cadvisor scrape job by default (#617)
+- For vmalert all replicas of alertmanager are added to notifiers (only if alertmanager is enabled) (#619)
+- Add `grafanaOperatorDashboardsFormat` option (#615)
+- Fix query expression for memory calculation in `k8s-views-global` dashboard (#636)
+- Bump version of Victoria Metrics components to `v1.93.3`
+- Bump version of operator helm-chart to `0.26.0`
```

Comment:
maybe to `0.26.*`? seems like will use `0.26.1` in this version.

Reply:
No, unfortunately it doesn't work that way and at the time of release the version that is in the release - 0.26.0 - will be used for build.